smartcontract/service/neovm/config.go

```diff
@@ -191,5 +193,7 @@ func initGAS_TABLE() *sync.Map {
 	m.Store(RUNTIME_BASE58TOADDRESS_NAME, RUNTIME_BASE58TOADDRESS_GAS)
 	m.Store(RUNTIME_ADDRESSTOBASE58_NAME, RUNTIME_ADDRESSTOBASE58_GAS)
 
+	m.Store(RUNTIME_GETCURRENTBLOCKHASH_NAME, RUNTIME_VERIFYMUTISIG_GAS)
```

Comment:
what is this?

Reply:
verify mutil sig will spent extra gas fee